sips/sip-021/sip-021-nakamoto.md

```diff
@@ -0,0 +1,556 @@
+
+In the act of producing a `TenureChange` transaction, the Stackers also internally agree to no longer sign the current miner's blocks. Thus, the act of producing a `TenureChange` atomically transfers block-production responsibilities from one miner to another. The new miner cannot orphan recently-confirmed blocks from the old miner because the `TenureChange` transaction contains within it the most recently confirmed blockId as identified by the same Stackers that produced the `TenureChange`.  As a result, any new blocks that fail to build upon the latest Stacks block identified by the Signers will be verifiably invalid.
+
+![Figure 3][figure-3-asset]
+
+*Figure 3: Tenure change overview. When a new Bitcoin block arrives, Stackers begin the process of deciding the last block they will sign from miner N. When they reach quorum, they make this data available for download by miners, and wrap it in a WSTS-signed specially-crafted data payload. This information serves as a record of the tenure change, and must be incorporated in miner N+1's tenure-start block. In other words, miner N+1 cannot begin producing Stacks blocks until Stackers inform it of block X -- the block from miner N that it must build atop. Stacks-on-Bitcoin transactions are applied by miner N+1 for all Bitcoin blocks in sortitions N and earlier when its tenure begins.*
+
+The `TenureChange` transaction encodes the following data:
+
+|Name|Description|Representation|
+|-|-|-|
+|tenure consensus hash|Consensus hash of this tenure.  Corresponds to the sortition in which the miner of this block was chosen.  It may be the case that this miner's tenure gets extended across subsequent sortitions; if this happens, then this `consensus hash` value remains the same as the sortition in which the winning block-commit was mined.|20 bytes|
+|previous tenure consensus hash|Consensus hash of the previous tenure.  Corresponds to the sortition of the previous winning block-commit.|20 bytes|
+|burn view consensus hash|Current consensus hash on the underlying burnchain.  Corresponds to the last-seen sortition.|20 bytes|
+|previous tenure end|The index block hash of the last Stacks block from the previous tenure.|32 bytes|
+|previous tenure blocks|The number of blocks produced since the last sortition-linked tenure.|4 bytes, big-endian|
+|cause|A flag to indicate the cause of this tenure change<br>- `0x00` indicates that a sortition occurred, and a new miner should begin producing blocks.<br>- `0x01` indicates that the current miner should continue producing blocks. The current miner’s tenure execution budget is reset upon processing this transaction.|1 byte|
+|pubkey hash|The ECDSA public key hash of the current tenure.|20 bytes|
+|signature|The Stacker signature.|65 bytes|
```

Comment:
Same as above (Schnorr signatures are 64 bytes)

```suggestion
|signature|The Stacker signature.|64 bytes|
```

Reply:
The `signature` and the `signers` bitmap have been removed.

Reply:
Removed.